rfc/rfc021-vp_token-grant-type.md

```diff
@@ -132,20 +136,67 @@ The second paragraph describes the requirements that apply to JWT encoded VP's.
 5. The `verificationMethod` field of the Proof MUST be a DID URL.
 6. The `holder` field if present MUST match the `credentialSubject.id` field from all the Verifiable Credentials that are used to request the access token.
 7. The `verificationMethod` field of the proof MUST match the `credentialSubject.id` field from all the Verifiable Credentials that are used to request the access token.
+8. The `created` field of the proof MUST be present and contain a valid timestamp. It MUST be before the current time.
+9. The `expires` field of the proof MUST be present and contain a valid timestamp. It MUST be after the current time.
+10. The difference between the `expires` and `created` fields MUST be equal or less than 5 seconds.
+
+### 4.4 Preventing Token Replay
+
+The Authorization Server MUST reject any request that uses the unique value (`jti` or `challenge`) that has been used before.
```

Comment:
why must `challenge` be unique, if `jti` already is?

Reply:
jti is used for jwt's, challenge for JsonWebSignature2020